boards.txt

```diff
@@ -715,6 +731,11 @@ esp32.menu.DebugLevel.debug.build.code_debug=4
 esp32.menu.DebugLevel.verbose=Verbose
 esp32.menu.DebugLevel.verbose.build.code_debug=5
 
+esp32.menu.EraseFlash.none=Only Sketch
+esp32.menu.EraseFlash.none.upload.erase_cmd=
+esp32.menu.EraseFlash.all=All Flash Contents
+esp32.menu.EraseFlash.all.upload.erase_cmd=-e
+
```

Comment:
If the comment above is accepted, please implement as a single option in Menu:

"Erase all Flash before uploading the sketch:" "Yes" or "No"